src/vertex_array_object.rs

```diff
@@ -467,15 +467,36 @@ unsafe fn bind_attribute(ctxt: &mut CommandContext, program: &Program,
                                                  stride as i32,
                                                  (buffer_offset + offset) as *const libc::c_void),
 
-                gl::DOUBLE | gl::DOUBLE_VEC2 | gl::DOUBLE_VEC3 | gl::DOUBLE_VEC4 |
-                gl::DOUBLE_MAT2 | gl::DOUBLE_MAT3 | gl::DOUBLE_MAT4 |
-                gl::DOUBLE_MAT2x3 | gl::DOUBLE_MAT2x4 | gl::DOUBLE_MAT3x2 |
-                gl::DOUBLE_MAT3x4 | gl::DOUBLE_MAT4x2 | gl::DOUBLE_MAT4x3 =>
+                gl::DOUBLE =>
                     ctxt.gl.VertexAttribLPointer(attribute.location as u32,
                                                  elements_count as gl::types::GLint, data_type,
                                                  stride as i32,
                                                  (buffer_offset + offset) as *const libc::c_void),
 
+                gl::FLOAT => {
+                    ctxt.gl.VertexAttribPointer(attribute.location as u32,
+                                                elements_count as gl::types::GLint, data_type, 0,
+                                                stride as i32,
+                                                (buffer_offset + offset) as *const libc::c_void)
+                },
+
+                gl::FLOAT_MAT4 => {
```

Comment:
Execution gets here for a `mat4`, and I tried to match up the arguments to `VertexAttribPointer` so that they look like the ones for `gl::FLOAT`. But it still doesn't render.